src/engine/ruleset/decoders/fortinet/fortinet-fortimail.yml

```diff
@@ -0,0 +1,167 @@
+---
+name: decoder/fortinet-fortimail/0
+
+metadata:
+  module: fortinet
+  title: Decoder for Fortinet fortimail
+  description: Decoder for Fortinet fortimail
+  version: [6.0.x, 6.2.x]
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/09
+  references:
+    - https://docs.fortinet.com
+
+sources:
+  - decoder/queue-localfile/0
+
+definitions:
+  def_parser_header: date=<~log.date> time=<~log.time> device_id=<observer.serial_number> log_id=<event.code> log_part=<~log.rsa.misc.reference_id1>
+  def_parser_type_subtype_pri: type=<event.action> subtype=<~log.rsa.internal.messageid> pri=<log.level>
+  def_parser_type_rsa_subtype: $def_parser_type_rsa_subtype
+  def_parser_type_pri: type=<event.action> pri=<log.level>
+
+parse:
+  logpar:
+    # date=2018-1-27 time=05:21:06 device_id=atcupid log_id=onse log_part=psa type=virus_file-signature pri=high destla to="fugitse" src=[10.12.86.130] session_id=dese msg="Attachment file (duntutla) has sha1 hash value: lamco"
```

Comment:
Check these logs are being processed by the fortimanager decoder instead of the fortimail decoder.